src/pages/publication/PublicationForm.tsx

```diff
@@ -158,34 +161,38 @@ const PublicationForm: FC<PublicationFormProps> = ({ uppy = createUppy(shouldAll
 
   return (
     <StyledPublication>
-      <Formik
-        enableReinitialize
-        initialValues={initialValues}
-        validationSchema={validationSchema}
-        onSubmit={(values: Publication) => savePublication(values)}
-        validateOnChange={false}>
-        {({ values }: FormikProps<Publication>) => (
-          <Form>
-            <PublicationFormTabs tabNumber={tabNumber} handleTabChange={handleTabChange} />
-            {tabNumber === 0 && (
-              <DescriptionPanel goToNextTab={goToNextTab} savePublication={() => savePublication(values)} />
-            )}
-            {tabNumber === 1 && (
-              <ReferencesPanel goToNextTab={goToNextTab} savePublication={() => savePublication(values)} />
-            )}
-            {tabNumber === 2 && (
-              <ContributorsPanel goToNextTab={goToNextTab} savePublication={() => savePublication(values)} />
-            )}
-            {tabNumber === 3 && <FilesAndLicensePanel goToNextTab={goToNextTab} uppy={uppy} />}
-
-            {tabNumber === 4 && (
-              <TabPanel ariaLabel="submission">
-                <SubmissionPanel savePublication={() => savePublication(values)} />
-              </TabPanel>
-            )}
-          </Form>
-        )}
-      </Formik>
+      {isLoading ? (
+        <Progress />
```

Comment:
må styles, men lager egen sak på det (NP-587), siden det trolig medfører endringer for stiling av tab panels også